src/config.h

```diff
@@ -157,3 +157,33 @@
 #ifndef MG_ENABLE_PROFILE
 #define MG_ENABLE_PROFILE 0
 #endif
+
+#ifndef MG_ENABLE_DRIVER_INIT
```

Comment:
driver... what driver ? flash driver ? tcpip driver ? future driver ?
This is tempting but I think we have to prefix with TCPIP, as the drivers themselves are, if not for clarity just for consistency